library-no-op/src/main/kotlin/com/chuckerteam/chucker/api/Chucker.kt

```diff
@@ -19,4 +22,11 @@ public object Chucker {
     public fun dismissNotifications(context: Context) {
         // Empty method for the library-no-op artifact
     }
+
+    @JvmStatic
+    public fun writeTransactions(
+        context: Context,
+        maxTransactions: Long?,
+        startTimestamp: Long?,
+    ): Uri? = null
```

Comment:
I think this should be moved to `ChuckerCollector`

Reply:
I have updated this to be inside ChuckerCollector, but it seems slightly odd to me - (as a consumer) why do I have to instantiate a class that is about handling sending/receiving http messages in order to dump the saved transactions to a file?

Reply:
The documentation says:

```
 * The collector responsible of collecting data from a [ChuckerInterceptor] and
 * storing it/displaying push notification. 
```

While the `Chucker` class you used originally is just the entry point of our library. As you want to do an advanced operation, I think it's reasonable to ask the user to instanciate and extra object. This will not pollute the `Chucker` interface making it a god object in the long run.